bricks/presentational-bricks/src/general-preview-image/index.tsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { BrickWrapper, property, UpdatingElement } from "@next-core/brick-kit";
+import { GeneralPreviewImage } from "./GeneralPreviewImage";
+
+/**
+ * @id presentational-bricks.general-preview-image
+ * @author Albert Jie
+ * @history
+ * 1.x.0: 新增构件 `presentational-bricks.general-preview-image`
+ * @docKind brick
+ * @noInheritDoc
+ */
+export class GeneralPreviewImageElement extends UpdatingElement {
+  /**
+   * @default -
+   * @required
+   * @description 图像高度
+   */
+  @property({
+    attribute: false,
+  })
+  height: string | number;
+
+  /**
+   * @default -
+   * @required
+   * @description 图像宽度
+   */
+  @property({
+    attribute: false,
+  })
+  width: string | number;
+
+  /**
+   * @default -
+   * @required
+   * @description 图片地址
+   */
+  @property()
+  src: string;
+
+  /**
+   * @default -
+   * @description 多图预览时所有图片地址
+   */
+  @property()
+  srcList: string[];
```

Comment:
`@property()` 默认为 `type: String`，不支持 `string[]`，需要改为：

```suggestion
  @property({ attribute: false })
  srcList: string[];
```